.rubocop.yml

```diff
@@ -98,6 +98,9 @@ Style/StringLiteralsInInterpolation:
 Style/SymbolArray:
   EnforcedStyle: brackets
 
+Metrics/BlockLength: 
```

Comment:
Why are you increasing this?

Reply:
I get and error by rubocop on schema.rb file by this rule

Reply:
Just wat I was expecting
Instead of avoiding the issue, you should fix it
Eventually you schema will wrong so big it'll reach 1455 lines. That doesn't mean every once in a while you need to come back to rubicon config and increase the BlockLength number. You don't need to worry about how big your schema is, that being said, let's instead ignore the file

Reply:
https://docs.rubocop.org/rubocop/configuration.html#cop-specific-include-and-exclude

Reply:
You're right!, Fixed.